.github/workflows/nightly.yaml

```diff
@@ -6,7 +6,7 @@ on:
     - cron: "0 1 * * *"
 
 env:
-  PYTHON_VERSION: 3.8
+  PYTHON_VERSION: 3.10
```

Comment:
Causes nightly to fail https://github.com/home-assistant/frontend/runs/7420855703?check_suite_focus=true

Version should be stringified because trailing 0. Fix here https://github.com/home-assistant/frontend/pull/13235